### DISCUSSION TITLE
Member of claim renew the purge count time

### DISCUSSION THREAD

**Discussion by comonier** [GriefPrevention/GriefPrevention#932] [Ideas]
Advantage of the RP over the GP, members added to the RP refresh the terrain, to avoid the purge, in the GP only the owner can refresh the claim purge time.
![gp1](https://user-images.githubusercontent.com/19571541/89196979-629a5380-d581-11ea-8053-bc1e4b806a64.png)
on GriefPrevention i did a claim with account reinomoc and add account comonier on memberlist but i can do login only with comonier and the claim will not refresh to renew purge time.
![rp1](https://user-images.githubusercontent.com/19571541/89196982-63cb8080-d581-11ea-9c33-940891b00d60.png)
on Redprotect (other plugin) that can be possible cause the owner include when a member of the claim do login renew the purge count time. Just a suggest to implement on GriefPrevention.
in english it says [Last visit of owner or member: Today] Purge Enabled: Yes - recounting.

I know that each plugin has its characteristics and I love GriefPrevention for being light and easy to use, not to mention that it has been around for many years and everyone knows and knows how to use the golden shovel plugin, but it would be really cool to have this option in the settings: Allow members of the trustlist to renew the purge time of the claim.

**Comment by RoboMWM** [comment]
I think this is a duplicate request.

GP only checks owners instead of all trustees, and I think there was a reason for this provided by Big_Scary himself. Additionally, trustees wouldn't be able to resize or remove the claim if needed.

IMO, a better solution would be to transfer the claim to someone else on expire.

**Reply by comonier** [reply]
> I think this is a duplicate request.
> 
> GP only checks owners instead of all trustees, and I think there was a reason for this provided by Big_Scary himself. Additionally, trustees wouldn't be able to resize or remove the claim if needed.
> 
> IMO, a better solution would be to transfer the claim to someone else on expire.

not better if you wont lose the owner of the claim. a member renewing that garants you not lose the owner, example, you play on a server for years, you builded a lot of thinks, you need to stop to play on server cause some personal problem, but on that server the owner setup purge to 20 days, and you will lose that 100% garanted cause you cant play for some reason, if you add someone who play always on server as /accesstrust that play cant remove blocks on your claim, but will still doing login on server and will refresh the claim.

**Reply by RoboMWM** [reply]
there are other options in the default config that allow for this scenario (see total claimblocks and bonus blocks).
